### PR TITLE
Monthly

### PR DESCRIPTION
**Pull request by ssjunnebo**

### CODE REVIEW

**Review by aanil — APPROVED** [review]
👍

File: roles/arteria-delivery-ws/defaults/main.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@
 #
 # This will set corresponding paths and use the appropriate port.
 arteria_delivery_repo: https://github.com/arteria-project/arteria-delivery.git
-arteria_delivery_version: v2.6.3
+arteria_delivery_version: v3.0.0-rc2
```
Comment (b97pla) [inline roles/arteria-delivery-ws/defaults/main.yml:10]: @Aratz is this the correct version for production? ☝️

**Review by b97pla — APPROVED** [review]
:+1: looks good to me but I think it's best if @Aratz can comment on the Arteria-delivery service version before merging.

File: roles/arteria-delivery-ws/defaults/main.yml (hunk 1 above)
Comment (b97pla) [reply roles/arteria-delivery-ws/defaults/main.yml:10]: ok, then I guess all is well. thanks!

**Pull request merged**